presto-main-base/src/main/java/com/facebook/presto/operator/TableFinishConfig.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+import com.facebook.airlift.units.DataSize;
+import jakarta.validation.constraints.NotNull;
+
+import static com.facebook.airlift.units.DataSize.Unit.MEGABYTE;
+
+public class TableFinishConfig
```

Comment:
I couldn't find a better config file to put this config in, so I made my own. I'm open to suggestions on where else to put this!